package.json

```diff
@@ -12,29 +12,29 @@
   },
   "main": "server.coffee",
   "dependencies": {
-    "americano": "0.4.2",
-    "async": "^0.9.0",
-    "cozy-notifications-helper": "^1.0.2",
+    "americano": "0.4.5",
+    "async": "1.5.2",
+    "cozy-notifications-helper": "1.1.0",
     "cozydb": "0.1.10",
     "gdata-js": "3.0.1",
-    "gm": "1.17.0",
-    "googleapis": "2.0.2",
-    "imagemagick-stream": "^2.0.0",
-    "jade": "1.9.1",
-    "lodash": "^3.8.0",
-    "moment-timezone": "^0.3.1",
-    "node-polyglot": "^0.4.1",
-    "printit": "0.1.15",
+    "gm": "1.22.0",
+    "googleapis": "4.0.0",
+    "imagemagick-stream": "4.0.1",
```

Comment:
This version is not compatible with node 0.10, see https://github.com/cozy-labs/konnectors/commit/fc91e5318312d0b44095b476769a127aaf808756